hack/build-local-images.py

```diff
@@ -138,13 +148,10 @@ def image_rebuild_requested(image):
     suffix explicitly or does not provide
     any explicit requests.
     """
-    return len(sys.argv) == 1 or (
-        len(sys.argv) > 1 and (
-            image in sys.argv or
-            full_name(image) in sys.argv
-        )
-    )
-
+    return (len(sys.argv) == 1
```

Comment:
nit: as this gets more unwieldy maybe it's good to start using explanatory booleans:

```py
implicitly_triggered = len(sys.argv) == 1 and image_config[image].get("enable_default", True)
explicitly_triggered = len(sys.argv) > 1 and (image in sys.argv or full_name(image) in sys.argv)
return implictly_triggered or explicitly_triggered
```